public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -97,7 +98,7 @@ export function panelMenuBehavior(menu: VizPanelMenu) {
       text: t('panel.header-menu.duplicate', `Duplicate`),
       onClick: () => {
         DashboardInteractions.panelMenuItemClicked('duplicate');
-        dashboard.duplicatePanel(panel);
+        libraryPanel ? dashboard.duplicateLibraryPanel(libraryPanel) : dashboard.duplicatePanel(panel);
```

Comment:
Looking at this, what do you think if we only have `dashboard.duplicatePanel(libraryPanel)` in the dashboard scene and this distinction is managed internally?

Reply:
Yeah, maybe. I mean - library panels also respect panel repeat options, which currently ain't handled. I suppose with that in place the difference between these two functions will be quite cosmetic